internal/provisioner/model/model.go

```diff
@@ -673,6 +679,7 @@ func New(cfg Config) *Contour {
 		},
 		Spec: ContourSpec{
 			ContourReplicas: cfg.Replicas,
+			LogLevel:        contourv1alpha1.DebugLog,
```

Comment:
I think we should leave the default as info, switching to debug logging should be opt-in.

Reply:
i find the New only used in unittests, so set the `LogLevel` to `DebugLog`, I will change it